docker-compose.yml

```
@@ -0,0 +1,64 @@
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U user -d mydb"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+volumes:
+  postgres_data:
+  chromadb_data:
+  searxng_data: 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)